.github/workflows/feedstock_pr_create.yaml

```diff
@@ -0,0 +1,55 @@
+on:
+  workflow_dispatch:
```

Comment:
Is the idea here that once the pypi release is done we would manually trigger this workflow with the new version to create the feedstock PR?

Reply:
I have updated `release.md` with the instructions

Reply:
Could the `release` github action kick off this workflow after it finishes the pypi upload?

Reply:
@rwedge What about this?
https://github.com/alteryx/featuretools/pull/2181/commits/f56f48fee51a375c46a08074185484077cff1916